MathLib/LinAlg/UnifiedMatrixSetters.cpp

```diff
@@ -89,9 +89,27 @@ void setVector(Eigen::VectorXd& v, MatrixVectorTraits<Eigen::VectorXd>::Index co
 namespace MathLib
 {
 
-double norm(PETScVector const& x)
+double norm(PETScVector const& x, MathLib::VecNormType nmtype = MathLib::VecNormType::NORM2)
```

Comment:
changing the behavior is not a part of this PR, though I agree having a default value here is no help.